analyzers/src/SonarAnalyzer.VisualBasic/Rules/Hotspots/DeliveringDebugFeaturesInProduction.cs

```diff
@@ -43,6 +43,12 @@ public sealed class DeliveringDebugFeaturesInProduction : DeliveringDebugFeature
                 context.Node.FirstAncestorOrSelf<StatementSyntax>() is { } invocationStatement
                 && invocationStatement.Ancestors().Any(node => IsDevelopmentCheck(node, context.SemanticModel));
 
+        protected override TrackerBase<SyntaxKind, InvocationContext>.Condition IsInDevelopmentContext() =>
+            context => context.Node
+                              .Ancestors()
+                              .OfType<ClassBlockSyntax>()
+                              .Any(classBlockSyntax => classBlockSyntax.ClassStatement.Identifier.Text == StartupDevelopment);
```

Comment:
This is the case where `x` will make it shorter and more readable. It's clear that it's `ClassBlockSyntax` from the previous line.
```suggestion
                              .Any(x => x.ClassStatement.Identifier.Text == StartupDevelopment);
```